cluster/ci/config/prow/config.yaml

```diff
@@ -29,6 +29,7 @@ tide:
   queries:
   - repos:
     - openshift/image-registry
+    - openshift/online-hibernation
     labels:
     - lgtm
     - approved
```

Comment:
@stevekuznetsov I think I want the 'approved' here, but can you clarity what that means? I'm thinking it's an extra hurdle b4 auto-merging, right? I also added 'approve' to the plugins.yaml stanza.  They go together, right?

Reply:
Yes, you need `lgtm` plus the `approved` label added by the `approve` plugin below.

Reply:
@sallyom in order to use this you'll need to create OWNERS files in your repository for the tools to choose approvers from